src/js/wand.js

```diff
@@ -50,12 +53,20 @@ var Wand = (function(wand) {
     wand.opts.nodes.forEach(function(node) {
 
       if (hasValidApiCallbackFns(node) === false) {
-        throw noSuchCallbackError;
+        wand.notifications.add({
+          text: 'We could not find a callback function that you asked for!',
+          type: 'alert'
+        });
+        throw wand.errors.noSuchCallbackError;
```

Comment:
Do we want to incorporate the `throw` piece into `notifications` so we don't have to run two separate functions for one error? Something like:

```
wand.notifications.add({
    text: 'We could not find a callback function that you asked for!',
    type: 'alert',
    console: wand.errors.noSuchCallbackError
});
```